lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -256,7 +256,8 @@ def run(self):
 
 def _copy_css_file(app, exc):
     if exc is None and app.builder.format == 'html':
-        src = sphinxext._static_path / Path('plot_directive.css')
+        src = (Path(matplotlib.get_data_path()) /
+               'plot_directive/plot_directive.css')
```

Comment:
you can use `cbook._get_data_path("pd/pd.css")` which directly gives you a Path

Reply:
Done.